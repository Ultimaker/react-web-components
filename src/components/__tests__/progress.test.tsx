// Copyright (c) 2018 Ultimaker B.V.
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import ProgressBar from '../progress_bar';

describe('The ProgressBar component', () => {
    let props;
    let wrapper;

    beforeEach(() => {
        props = {
            progressPercentage: 50,
        };
        wrapper = shallow(<ProgressBar {...props} />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
