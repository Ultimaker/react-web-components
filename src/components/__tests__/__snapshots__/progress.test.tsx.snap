// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The ProgressBar component should render 1`] = `
<div
  className="progress-bar"
>
  <span
    className="progress-bar__progress-indicator"
    style={
      Object {
        "height": "0.6rem",
        "width": "50%",
      }
    }
  />
  <span
    className="progress-bar__empty"
    style={
      Object {
        "height": "0.6rem",
        "width": "50%",
      }
    }
  />
</div>
`;
